### PR TITLE
Move Web/API/Houdini to Web/API/Houdini_APIs | fix the reference bug

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

in https://developer.mozilla.org/en-US/docs/Web/API, the Houdini is a broken link:

image

this is because the *overview* key does not match the summary page

image

sso choose to move the api summary page, also match the behavior of other apis (especially Sensor APIs)

### CODE REVIEW

File: files/en-us/glossary/houdini/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ While many of the features Houdini enables can be created with JavaScript, inter
 
 ## See also
 
-- [Houdini](/en-US/docs/Web/API/Houdini)
+- [Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/glossary/houdini/index.md:17]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/api/css_object_model/index.md [hunk 2]
```diff
@@ -112,4 +112,4 @@ All these features have been added little by little over the years to the differ
 ## See also
 
 - [Document Object Model (DOM)](/en-US/docs/Web/API/Document_Object_Model)
-- [Houdini](/en-US/docs/Web/API/Houdini)
+- [Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/api/css_object_model/index.md:115]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/api/css_painting_api/index.md [hunk 3]
```diff
@@ -157,4 +157,4 @@ While you can't play with the worklet's script, you can alter the custom propert
 
 - [Using the CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API/Guide)
 - [CSS Typed Object Model API](/en-US/docs/Web/API/CSS_Typed_OM_API)
-- [CSS Houdini](/en-US/docs/Web/API/Houdini)
+- [CSS Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/api/css_painting_api/index.md:160]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/api/css_properties_and_values_api/index.md [hunk 4]
```diff
@@ -54,4 +54,4 @@ The same registration can take place in [CSS](/en-US/docs/Web/CSS) using the {{c
 - [Using the CSS properties and values API](/en-US/docs/Web/API/CSS_Properties_and_Values_API/guide)
 - [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API)
 - [CSS Typed Object Model](/en-US/docs/Web/API/CSS_Typed_OM_API)
-- [CSS Houdini](/en-US/docs/Web/API/Houdini)
+- [CSS Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/api/css_properties_and_values_api/index.md:57]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/css/@property/index.md [hunk 5]
```diff
@@ -136,6 +136,6 @@ For item three, the `--item-size` value gets set to `1000px`. While `1000px` is
 - [CSS Properties and Values API](/en-US/docs/Web/API/CSS_Properties_and_Values_API)
 - [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API)
 - [CSS Typed Object Model](/en-US/docs/Web/API/CSS_Typed_OM_API)
-- [CSS Houdini](/en-US/docs/Web/API/Houdini)
+- [CSS Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@property/index.md:139]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/css/@property/inherits/index.md [hunk 6]
```diff
@@ -78,4 +78,4 @@ window.CSS.registerProperty({
 - [CSS Properties and Values API](/en-US/docs/Web/API/CSS_Properties_and_Values_API)
 - [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API)
 - [CSS Typed Object Model](/en-US/docs/Web/API/CSS_Typed_OM_API)
-- [CSS Houdini](/en-US/docs/Web/API/Houdini)
+- [CSS Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@property/inherits/index.md:81]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/css/@property/initial-value/index.md [hunk 7]
```diff
@@ -77,4 +77,4 @@ window.CSS.registerProperty({
 - [CSS Properties and Values API](/en-US/docs/Web/API/CSS_Properties_and_Values_API)
 - [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API)
 - [CSS Typed Object Model](/en-US/docs/Web/API/CSS_Typed_OM_API)
-- [CSS Houdini](/en-US/docs/Web/API/Houdini)
+- [CSS Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@property/initial-value/index.md:80]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/css/@property/syntax/index.md [hunk 8]
```diff
@@ -102,4 +102,4 @@ window.CSS.registerProperty({
 - [CSS Properties and Values API](/en-US/docs/Web/API/CSS_Properties_and_Values_API)
 - [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API)
 - [CSS Typed Object Model](/en-US/docs/Web/API/CSS_Typed_OM_API)
-- [CSS Houdini](/en-US/docs/Web/API/Houdini)
+- [CSS Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/css/@property/syntax/index.md:105]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

File: files/en-us/web/css/css_properties_and_values_api/index.md [hunk 9]
```diff
@@ -84,7 +84,7 @@ The value of `--stop-color` is set to `cornflowerblue` at first, but when you ho
 
   - : Explains how to register custom properties in CSS and JavaScript, with hints on handling undefined and invalid values, fallbacks, and inheritance.
 
-- [CSS Houdini](/en-US/docs/Web/API/Houdini)
+- [CSS Houdini](/en-US/docs/Web/API/Houdini_APIs)
```
Comment (chrisdavidmills) [inline files/en-us/web/css/css_properties_and_values_api/index.md:87]:
```suggestion
- [Houdini APIs](/en-US/docs/Web/API/Houdini_APIs)
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks for the update @skyclouds2001. The change largely makes sense to me, although I've made some suggestions that would make the link text more consistent.

I also think we should get @estelle's blessing on this one — it was her that moved the Houdini landing page under `/APIs` recently, as part of the project to remove the `/Guide` tree.

**Review by chrisdavidmills — APPROVED** [review]
Let's just get this one merged; I don't think there is anything too controversial in here, thinking about it some more. Sorry for the delay.

**Comment by estelle** [comment]
Thanks!

**Pull request merged**